docs/source/quantization.rst

```diff
@@ -998,6 +998,30 @@ if ``dtype`` is ``torch.qint8``, make sure to set a custom ``quant_min`` to be `
 you call the `torch.ao.quantization.get_default_qconfig(backend)` or `torch.ao.quantization.get_default_qat_qconfig(backend)` function to get the default ``qconfig`` for
 ``fbgemm`` or ``qnnpack`` backend
 
+Frequently Asked Questions
+--------------------------
+
+1. How can I do quantized inference on GPU?:
+
+   We don't have official GPU support yet, but we have two protoypes:
+
+   i) PyTorch quantization + fx2trt lowering, inference in TensorRT (A100 and later GPUs): see examples in
+      `pytorch/TensorRT <https://github.com/pytorch/TensorRT/blob/master/py/torch_tensorrt/fx/test/quant/test_quant_trt.py>`_
+
+   ii) Integration with cudnn through native quantized cuda ops: see examples in `pytorch/test_quantized_op.py <https://github.com/pytorch/pytorch/blob/master/test/quantization/core/test_quantized_op.py#L4598>`_
+       this project is an early prototype and has been paused
+
+2. Where can I get ONNX support for my quantized model?:
+
+   Supporting export to onnx model is not a priority for PyTorch quantization, please open an issue in `GitHub - onnx/onnx <https://github.com/onnx/onnx>`_  when you encounter problems with ONNX,
```

Comment:
nit: IMHO, saying that it's not a priority sounds a little dismissive. Could we reword it?

Reply:
yeah, i've changed the language to omit that part